library/proc_macro/src/bridge/scoped_cell.rs

```diff
@@ -15,23 +14,6 @@ pub trait LambdaL: for<'a> ApplyL<'a> {}
 
 impl<T: for<'a> ApplyL<'a>> LambdaL for T {}
```

Comment:
`ApplyL` and `LamdaL` can probably be removed too by making `ScopeCell` always use `BridgeState` as content instead of being generic over the content.